### ISSUE TITLE
add an inverted highlight-unscanned view

### ISSUE THREAD

**Issue by technogeeky** [KSPModStewards/SCANsat#31]
From TODO.txt:
- an "inverted" display of sorts? Not as in inverted colors, but a map that greys out mapped areas while making non-mapped areas some other (preferably bright) color. (**Hremsfeld**)

This sounds like it would be trivial to accomplish with a color palette selector. In fact, given my experimentation with the Solarized color set (which is not colorblind friendly, I bet) which has a dark and a bright mode, I would say this should be a good test of the utility of color palette swapping.

**Comment by DMagic1** [comment]
Yes, should be simple, un-mapped areas are given a default color. Making this variable should be easy.
